bot/exts/valentines/valentine_zodiac.py

```diff
@@ -19,36 +22,117 @@ class ValentineZodiac(commands.Cog):
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
-        self.zodiacs = self.load_json()
+        self.zodiacs, self.zodiac_fact = self.load_comp_json()
 
     @staticmethod
-    def load_json() -> dict:
+    def load_comp_json() -> Tuple[dict, dict]:
         """Load zodiac compatibility from static JSON resource."""
-        p = Path("bot/resources/valentines/zodiac_compatibility.json")
-        with p.open(encoding="utf8") as json_data:
-            zodiacs = load(json_data)
-            return zodiacs
+        explanation_file = Path("bot/resources/valentines/zodiac_explanation.json")
+        compatibility_file = Path("bot/resources/valentines/zodiac_compatibility.json")
+        with explanation_file.open(encoding="utf8") as json_data:
+            zodiac_fact = json.load(json_data)
+            for zodiac_data in zodiac_fact.values():
+                zodiac_data['start_at'] = datetime.fromisoformat(zodiac_data['start_at'])
+                zodiac_data['end_at'] = datetime.fromisoformat(zodiac_data['end_at'])
+        with compatibility_file.open(encoding="utf8") as json_data:
+            zodiacs = json.load(json_data)
+        return zodiacs, zodiac_fact
```

Comment:
```suggestion
                zodiac_data['end_at'] = datetime.fromisoformat(zodiac_data['end_at'])

        with compatibility_file.open(encoding="utf8") as json_data:
            zodiacs = json.load(json_data)

        return zodiacs, zodiac_fact
```
Purely cosmetic apperance. I find it easier to read these with a bit of whitespace, as whitespace is free.